### PR TITLE
Serve random content from content directory

### PR DESCRIPTION
**Pull request by OpenNingia**
Here is a simple way to provide more content :cocktail:

### CODE REVIEW

**Comment by notjonmadden** [comment]
👍

**Comment by golergka** [comment]
Needs more quality random content. "Random content 1" and "Random content 2" doesn't cut it.

What about cat pictures though?

**Comment by pierrepln** [comment]
No content is better than bad content.

**Comment by OpenNingia** [comment]
We have the technology, now we need the users! :smile:

**Comment by golergka** [comment]
Might I suggest some entries from this list? Or may be the whole playlist

https://www.youtube.com/playlist?list=PL7XlqX4npddfrdpMCxBnNZXg2GFll7t5y

**Comment by hxjuneja** [comment]
seems like a good start 👍

**Comment by dskoda1** [comment]
This is a good way to at least show other contributors how to serve content 👍

**Comment by choogeboom** [comment]
👍

**Comment by Niximacco** [comment]
:+1:

**Comment by OpenNingia** [comment]
Sadly we should vote again, because I fixed some conflicts

File: chaos.py [hunk 1]
```diff
@@ -77,14 +62,19 @@ def start_http_server():
...
+    
+    os.system("pkill chaos_server")
+    subprocess.Popen([sys.executable, "server.py"], cwd=join(THIS_DIR, "server"))
+    
     log.info("starting http server")
     start_http_server()
-
-    log.info("entering event loop")
->>>>>>> Add fortune server on port 80
+
```
Comment (travishaby) [inline chaos.py]: @OpenNingia I think the log got dropped here `log.info("entering event loop")`

Comment (OpenNingia) [reply chaos.py]: It should be fixed, but there was a rebasing accident ( all these PR merged all the times made my fork collapse in merge errors xD ).

The history now is a little messy, If this is not voted I'll resubmit again from a cleaner branch.

**Comment by Bobsans** [comment]
👍

**Pull request closed without merging**